### DISCUSSION TITLE
Who is using

### DISCUSSION THREAD

**Discussion by chunfz** [elsa-workflows/elsa-core#4880] [Ideas]
It is recommended to create a "Who is using" issue to discuss the usage of the elsa-workflow project.

**Comment by sfmskywalker** [comment]
Makes sense. Perhaps it should also include a "call to action" to anyone to mention if they are using Elsa. What do you think?

**Comment by glime-ai** [comment]
To discuss the usage of the Elsa-workflow project and to share experiences or ask questions about who is using Elsa, you can create an issue on the project's GitHub repository. This is a common practice for open-source projects, as it allows users to connect with each other, share use cases, and potentially collaborate on solutions.

When creating such an issue, you should provide details about your use case, any specific questions you have, and what kind of information or feedback you are seeking from other users. It's also helpful to tag the issue appropriately and to be clear in your title that you're looking to discuss "Who is using" the project.

Here's a step-by-step guide on how to create the issue:

1. Go to the Elsa-workflows GitHub repository: https://github.com/elsa-workflows/elsa-core
2. Click on the "Issues" tab.
3. Click on the "New issue" button.
4. Choose a relevant issue template if available, or select "Open a blank issue."
5. Title your issue with something clear and descriptive, like "Discussion: Who is using Elsa Workflows in Production?"
6. In the issue description, explain that you're looking to discuss and learn about who is using Elsa Workflows, and provide any specific details or questions you have.
7. Submit the issue.

Once the issue is created, other users and contributors can join the discussion and provide their insights.
